src/app/window-handler.ts

```diff
@@ -2320,6 +2325,10 @@ export class WindowHandler {
   }
 
   public exitApplication = async (shouldRelaunch: boolean = true) => {
+    await appStats.sendAnalytics(
```

Comment:
@sbenmoussati @NguyenTranHoangSym As this logic takes ~5sec closing SDA will be delayed and it feels laggy.

I think to prevent this we can hide the main window and close it later once all the tasks are done. What do you guys think?